files/zh-cn/glossary/baseline/compatibility/index.md

```diff
@@ -2,16 +2,28 @@
 title: 基线（兼容性）
 slug: Glossary/Baseline/Compatibility
 l10n:
-  sourceCommit: d14b5a927958ec1ab9737f2f9fce0761a4c95c75
+  sourceCommit: 70aa041ebd7e31df76818bc8459f60659fe94c0c
 ---
 
 {{GlossarySidebar}}
 
-**基线**（Baseline）标识可跨浏览器使用的 Web 平台特性。基线可帮助你决定何时使用某项特性，告诉你该特性何时不太可能给网站访客带来兼容性问题。
+**基线**（Baseline）标识可跨浏览器使用的 Web 平台特性，包括 API、CSS 属性、JavaScript 语法。基线特性被描述为当前稳定浏览器中新提供的，或随着时间的推移可持续支持且广泛可用的特性。不符合基线标准的特性被描述为具有有限可用性。
 
-基线特性（如 API、CSS 属性集或 JavaScript 语法）可在 Chrome、Edge、Firefox 和 Safari 等多款知名且广泛使用的桌面和移动浏览器上一致运行。
+极限考虑以下浏览器的支持情况：
```

Comment:
```suggestion
基线考虑以下浏览器的支持情况：
```

Reply:
Oops really sorry for that ridiculous typo